.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+    es6: true,
+  },
+  parserOptions: { ecmaVersion: 8 }, // to enable features such as async/await
+  ignorePatterns: ['node_modules/*', 'lib/*', '!.prettierrc.js'],
+  extends: ['eslint:recommended'],
+  overrides: [
+    // This configuration will apply only to TypeScript files
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      parser: '@typescript-eslint/parser',
+      settings: { react: { version: 'detect' } },
+      env: {
+        browser: true,
+        node: true,
+        es6: true,
+      },
+      extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:react-hooks/recommended',
+        'prettier/@typescript-eslint',
+        'plugin:prettier/recommended',
```

Comment:
```suggestion
        'prettier',
```

This simpler config is the current suggested way to combine ESLint + TS + Prettier.

Reply:
From the ressources I've consulted to have prettier with eslint they suggest to do it with `plugin:prettier/recommended`
Here are some of the sources I've checked:
https://dev.to/caelinsutch/setting-up-a-typescript-nodejs-application-with-prettier-and-eslint-53jc
https://www.robertcooper.me/using-eslint-and-prettier-in-a-typescript-project
https://paulintrognon.fr/blog/typescript-prettier-eslint-next-js

Reply:
Those might be outdated. Here are the `typescript-eslint` instructions for this: https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#usage-with-prettier

Reply:
Sweet, thank you for providing an update ressource :)

Reply:
If you are good with the PR you can go ahead and merge it. Let me know if there's anything else